src/main/viz/WiggleTrack.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * @flow
+ */
+'use strict';
+
+import _ from 'underscore';
+import React from 'react';
+import d3utils from './d3utils';
+import dataCanvas from 'data-canvas';
+import shallowEquals from 'shallow-equals';
+import style from '../style';
+
+import WiggleCache from './WiggleCache';
+import BigWigSource from '../sources/BigWigDataSource';
```

Comment:
`import type BigWigSource from '../sources/BigWigDataSource';` and the lint error should be gone